mongodb/map.jinja

```diff
@@ -1,14 +1,19 @@
 {% set mongodb = salt['grains.filter_by']({
     'default': {
-        'pkg': 'mongodb',
-        'service': 'mongodb',
-        'conf_file': '/etc/mongodb/mongodb.conf',
-        'config': {
-            'key': 'value',
-        },
+      'key': '7F0CEB10',
+      'keyserver': 'keyserver.ubuntu.com',
+      'pkgs': [
+        {'mongodb-10gen': '*'},
```

Comment:
Why did you make this a dict? You should be able to just have a list of strings. The only reason for having a dict is if you need to specify the version.